files/justfiles/toggles.just

```diff
@@ -177,22 +177,81 @@ toggle-xwayland ACTION="prompt":
 
 # Toggle bash environment lockdown (mitigates LD_PRELOAD attacks)
 toggle-bash-environment-lockdown:
-    #!/usr/bin/bash
-    BASH_ENV_FILES=("$HOME/.bashrc" "$HOME/.bash_profile")
+    #!/bin/run0 /bin/bash
+    set -eu pipefail # very unfortunately getend passwd is not compatible with -o, it exits status 2 when it doesn't find every possible UID from 1000 to 1100 as valid. :(
+    [ "$UID" -eq 0 ] || { echo "This script must be run as root."; exit 1;} #almost certainly not needed, but doesn't hurt
+    user_string=$(getent passwd {1000..1100} | awk -F':' '{ print $1}' | tr '\n' ',' | sed 's/,*$//')
+    IFS=',' read -ra user_list <<< "$user_string"
+    if lsattr "/etc/profile" 2>/dev/null | awk '{print $1}' | grep -q 'i'; then
+    	change="unlocked"
+    else
+    	change="locked"
+    fi
+    b=$(tput bold) #This is the more correct way to bold and satisfies -eou pipefail
+    n=$(tput sgr0)
```

Comment:
`$b` and `$n` are globally sourced from https://github.com/secureblue/secureblue/blob/live/files/system/usr/lib/ujust/libformatting.sh. Redefining them in a single file is not very useful.